CODEOWNERS

```diff
@@ -221,6 +221,7 @@ homeassistant/components/input_datetime/* @home-assistant/core
 homeassistant/components/input_number/* @home-assistant/core
 homeassistant/components/input_select/* @home-assistant/core
 homeassistant/components/input_text/* @home-assistant/core
+homeassistant/components/input_timetable/* @home-assistant/core
```

Comment:
You should add @amitfin not home-assistant/core

Reply:
Done